app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,93 @@
+html {
+	font-size: 16px;
+	text-align: center;
+	font-family: sans-serif;
+}
+
+body {
+	background-color: #D3D3D3;
+}
+
+h1 {
+	padding-top: 40px;
+	font-size: 3rem;
+
+}
+
+header {
+	height: 150px;
+	background-color: gray;
+}
+
+main li {
+	list-style: none;
+	line-height: 30px;
+}
+
+main .main-list:hover {
+	background-color: gray;
+}
+
+a {
+	text-decoration: none;
+	color: black;
+}
+
+
+.button {
+	background-color: red;
+	width: 10%;
+	display: inline-block;
+	padding: 10px;
+	margin: 5px;
+	margin-right: auto;
+	margin-left: auto;
+/*	border-radius: 10px;*/
+
+}
+
+main {
+	clear: both;
+	width: 80%;
+	margin-left: auto;
+	margin-right: auto;
+	background-color: white;
+	padding-top: 15px;
+	padding-right: 30px;
+	padding-left: 10px;
+	padding-bottom: 15px;
+	overflow: auto;
+}
+
+.list {
+	background-color: #C0C0C0;
+	width: 40%;
+	margin-right: auto;
+	margin-left: auto;
+	padding-bottom: 10px;
+}
+
+h3 {
+	background-color: gray;
+	color: white;
+	padding: 7px;
+}
+
+.vendor-list {
+	margin-top: 10px;
+}
+
+.product-list-ul {
+	display: inline-block;
+}
+
+.product-list {
+	float:left;
+	padding: 10px;
+}
+
+.columns {
+	width: 50%;
+	float: left;
+	margin-top: 20px;
```

Comment:
Take a look at the settings in whichever text editor you are using to make sure tabs are set to 2 spaces.  This won't be true for all languages but is generic for RoR.